annalist.el

```diff
@@ -586,18 +586,16 @@ those settings for displaying recorded information instead of the defaults."
           (princ "* Global\n"))
         (annalist--print-headings name-store 0 settings local-name-store)))
     (when (or local-name-store name-store)
+      (require 'org)
       (with-current-buffer output-buffer-name
-        (let ((org-startup-folded annalist-org-startup-folded))
-          (org-mode))
         (read-only-mode -1)
+        (let ((org-startup-folded annalist-org-startup-folded)
+              (org-startup-align-all-tables t)
+              (org-startup-shrink-all-tables nil))
+          (ignore org-startup-align-all-tables org-startup-shrink-all-tables)
```

Comment:
Is there some warning without the ignore?

Reply:
Yes. I don't want them to be optimized out by the byte compiler.

Reply:
The byte compiler shouldn't complain about or optimize out dynamic variables (defvar/defcustom).  I think the normal way to handle cases like this where org is not needed/loaded initially is with a `defvar` for each variable with an empty initvalue.  Can you add those above the function instead?

Reply:
Sure. Just made those changes.